server/WebApiTemplate/Properties/launchSettings.json

```diff
@@ -3,7 +3,7 @@
     "EmeraldChameleonChat": {
       "commandName": "Project",
       "launchBrowser": true,
-      "launchUrl": "swagger",
+      //"launchUrl": "swagger",
```

Comment:
Was this breaking something?

Reply:
nope. It wasn't breaking anything. I can reverse this. The only way I can test the endpoint is by using it. I was getting annoyed to need to constantly remove the URL each time I launched debug.